app/services/create_member_for_api_members_controller.rb

```diff
@@ -27,6 +27,12 @@ def create
 
   private
 
+  def member_params
+    return @member_params if @member_params.present?
+    allowed = Member.new.attributes.keys.map(&:to_sym) + [:name]
+    @member_params = @params.select{ |k, v| allowed.include? k }
```

Comment:
I did it that way first, but I have to permit in the controller it anyway or it raises an `unpermitted parameter` error